bythepixel/kyrio-go2broadband
# Improve UX

Closes #3 
#### How should the reviewer test this?
- Navigate to http://dev.bythepixel.com/kyrio/ and ensure site follows QA from #3
